include/constants/field/map_load.h

```diff
@@ -0,0 +1,13 @@
+#ifndef POKEPLATINUM_CONSTANTS_FIELD_MAP_LOAD_H
+#define POKEPLATINUM_CONSTANTS_FIELD_MAP_LOAD_H
+
+enum {
+    MAP_LOAD_TYPE_OVERWORLD,
+    MAP_LOAD_TYPE_UNDERGROUND,
+    MAP_LOAD_TYPE_UNION,
+    MAP_LOAD_TYPE_COLOSSEUM,
+    MAP_LOAD_TYPE_BATTLE_TOWER,
+    MAP_LOAD_TYPE_MAX
```

Comment:
**question:** Will these ever be invoked in field scripts? Curious if we need to canonize these in `consts` definitions yet.

Reply:
unsure

Reply:
Fine to leave as is for now ,then.